enterprise/frontend/src/metabase-enterprise/whitelabel/index.js

```diff
@@ -77,6 +79,20 @@ if (hasPremiumFeature("whitelabel")) {
           options: getLoadingMessageOptions(),
           defaultValue: "doing-science",
         },
+        {
+          key: "help-link",
+          display_name: t`Help Link in the Settings menu`,
```

Comment:
should `settings` be lowercase?

```suggestion
          display_name: t`Help Link in the settings menu`,
```